README.md

```diff
@@ -89,9 +89,37 @@ deepstate-angr --num_workers 4 --output_test_dir out $DEEPSTATE/build/examples/I
 
 DeepState consists of a static library, used to write test harnesses, and command-line _executors_ written in Python. At this time, the best documentation is in the [examples](/examples) and in our [paper](https://agroce.github.io/bar18.pdf).
 
-## Fuzzing
+## Fuzzing with libFuzzer
+
+If you install a recent-enough clang, and run `cmake` when you install
```

Comment:
I cannot build the LIBFUZZER support with clang-4.0 (Ubuntu 16.04) because it fails to recognize the [fuzzer-no-link](https://reviews.llvm.org/D36600) option. It could be nice to specify which version could be used.